cuda_bindings/cuda/bindings/driver.pyx.in

```diff
@@ -53948,3 +53948,124 @@ def sizeof(objType):
     if objType == VdpOutputSurface:
         return sizeof(cydriver.VdpOutputSurface){{endif}}
     raise TypeError("Unknown type: " + str(objType))
+
+def _add_native_handle_getters() -> None:
```

Comment:
Another potential micro-optimization is to make it a cdef function:
```suggestion
cdef int _add_native_handle_getters() except?-1:
```
(need to make it return 0 on success to convince Cython 3 the error handling is sufficient)